exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -249,6 +249,9 @@ func logToCWLog(resourceAttrs map[string]interface{}, log plog.LogRecord, config
 			if metadata.LogStreamName != "" {
 				logStreamName = metadata.LogStreamName
 			}
+		} else if err != nil && config.EmfOnly {
```

Comment:
We also want to change it so that if it's EmfOnly, then if it can't get the log group/stream from the message, then we want to drop the message instead of sending it to the config log group/stream.

Reply:
You are right. I need to remove the err != nil from the else if statement. Will do.

Reply:
log stream name default is instance id and that comes from the ccwa config.